docs/pages/patterns/navbar.vue

```diff
@@ -82,29 +82,31 @@ quickLinksData: {
 
 `KtNavbar` injects the theme provided by the parent or globally.
 
-When `KtNavbarTheme` object is provided, it will replace the default theme.
+When `KtTheme` object is not provided, it will replace the default theme.
```

Comment:
I don’t understand this sentence since you added the `not`.

Reply:
when there is no `KtTheme` object provided. Kotti will use default theme (the dark blue one)